### ISSUE TITLE
implement parsedQueryToSearchFilters in the map

### ISSUE THREAD

**Issue by bakeiro** [Couchers-org/couchers#4733]
Idea of this ticket it to take it back, for more info check app/web/features/search/useRouteWithSearchFilters.ts

basically was a feature that was existing before, allows to write the state of the map in the URL, making it possible to share the URL to share a specific search with specific filters

**Comment by nabramow** [comment]
@bakeiro Is this still relevant? If so you could add a bit more details that would help a volunteer understand what they need to do?

**Comment by bakeiro** [comment]
yeah this ticket it's a draft, it's a not to me, leave it in 'ideas' for now :)

**Comment by bakeiro** [comment]
yes, I moved it to Couchers engineering board, since it's a missing feature of the map, I just commented out in my refactor, the idea of this ticket it to take it back, for more info check `app/web/features/search/useRouteWithSearchFilters.ts`

**Comment by bakeiro** [comment]
I've written more details, I guess now it's fine to move it to the engineering backlog?

**Comment by bakeiro** [comment]
update:
explained the ticket + moved to the tech debt
